plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMigrateProcedure.java

```diff
@@ -92,7 +93,6 @@ public void testMigratePartitionedTable()
         String icebergTableName = "iceberg.tpch." + tableName;
 
         assertUpdate("CREATE TABLE " + hiveTableName + " WITH (partitioned_by = ARRAY['part_col']) AS SELECT 1 id, 'part1' part_col", 1);
-        assertQueryFails("SELECT * FROM " + icebergTableName, "Not an Iceberg table: .*");
```

Comment:
Why are these statements removed?

Reply:
Please see https://github.com/trinodb/trino/pull/17391#discussion_r1206552598

Reply:
However, the change is unrelated to "Support timestamp(3) type in Iceberg migrate procedure". You don't need to remove the existing check.

Reply:
I reverted the unrelated changes